index.js

```diff
@@ -73,7 +73,7 @@ module.exports = async (target, options) => {
 		if (options.app) {
 			command = options.app;
 		} else {
-			const useSystemXdgOpen = process.versions.electron || process.platform === 'android';
+			const useSystemXdgOpen = process.versions.electron || process.platform === 'android' || __dirname === '/';
```

Comment:
`__dirname === '/'` should be moved to a variable with a readable name.

Reply:
Should look better now